### PR TITLE
Cleanup Processors

### PR DESCRIPTION
**Pull request by scothis**
- RequestProcessor -> Handler
  - plus extreme simplification to one Configuration and Route
- StreamProcessor -> Processor

### CODE REVIEW

File: pkg/reconciler/v1alpha1/handler/resources/configuration.go [hunk 1]
```diff
@@ -20,28 +20,27 @@ import (
...
 	requestv1alpha1 "github.com/projectriff/system/pkg/apis/request/v1alpha1"
-	"github.com/projectriff/system/pkg/reconciler/v1alpha1/requestprocessor/resources/names"
+	"github.com/projectriff/system/pkg/reconciler/v1alpha1/handler/resources/names"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// MakeConfiguration creates a Configuration from an RequestProcessor object.
-func MakeConfiguration(rp *requestv1alpha1.RequestProcessor, i int) (*knservingv1alpha1.Configuration, error) {
-	rpsi := rp.Spec[i]
+// MakeConfiguration creates a Configuration from an Handler object.
```
Comment (markfisher) [inline pkg/reconciler/v1alpha1/handler/resources/configuration.go]: "a Handler"

File: pkg/reconciler/v1alpha1/handler/resources/route.go [hunk 2]
```diff
@@ -17,30 +17,39 @@ limitations under the License.
...
-	"github.com/projectriff/system/pkg/reconciler/v1alpha1/requestprocessor/resources/names"
+	"github.com/projectriff/system/pkg/reconciler/v1alpha1/handler/resources/names"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// MakeFunction creates a Function from an RequestProcessor object.
-func MakeFunction(rp *requestv1alpha1.RequestProcessor, i int) (*buildv1alpha1.Function, error) {
-	if rp.Spec[i].Build.Function == nil {
-		return nil, nil
+// MakeRoute creates a Route from an Handler object.
```
Comment (markfisher) [inline pkg/reconciler/v1alpha1/handler/resources/route.go]: "a Handler"

**Comment by markfisher** [comment]
LGTM

**Comment by scothis** [comment]
rebased into master as edb02f2a0eaa64369b146c27e34863277934eb55

**Pull request closed without merging**